WordPress/src/main/res/values/strings.xml

```diff
@@ -3650,6 +3650,10 @@
     <string name="blogging_reminders_notify_me">Notify me</string>
     <string name="blogging_reminders_update">Update</string>
     <string name="blogging_reminders_done">Done</string>
+    <string name="blogging_reminders_epilogue_title">All set!</string>
+    <string name="blogging_reminders_epilogue_body_days">You\'ll get reminders to blog %1$s on %2$s.</string>
```

Comment:
Instead of reusing the `R.array.blogging_goals_count` array to compose this string, we should probably use numbers directly, as it is defined in our design spec:

<img width="360" alt="Success sheet" src="https://user-images.githubusercontent.com/830056/122465708-cc9bcb80-cf8e-11eb-92e8-01bf4e7a66d2.png">

I know you are probably going to handle this in an upcoming PR, but I just want to make sure we're on the same page.

Reply:
I had it before like that **2** times.  

I've updated it to be consistent with the summary we show now in Site settings.  Also, it reads better For e.g. **1** time a week needs to read **once** a week or **2** times a week as **twice** a week and the rest as three times...

Yeah, if the design review suggests to go with earlier design then I'll revert it back.